Week-9/src/App.js

```diff
@@ -0,0 +1,63 @@
+import "./styles.css";
+import React, { useState } from "react";
+
+export default function App() {
+  const [list, setList] = useState([]);
+  const [item, setItem] = useState("");
+
+  handleInput = (e) => {
+    setItem(e.target.value);
+  };
+
+  onAdd = (e) => {
+    e && e.preventDefault();
+    let lists = list;
```

Comment:
make a copy of list first and then items , currently we are making changes to state as both lists and list have same reference